### PR TITLE
circleci: introduce workflow to release a chart to ChartMuseum

### PR DESCRIPTION
**Pull request by asdwsda**
New workflow can be triggered by pushing a tag in the following format: <chart_name>/<version>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,6 +3,26 @@ version: 2.0
...
+            pwd
+            find -H . \
+                -maxdepth 1 \
+                -type d \
+                ! -path . \
+                ! -path ./.gitignore \
+                ! -path ./.git \
+                ! -path ./.circleci \
+                -print \
+                    | xargs helm lint
```
Comment (orymate) [inline .circleci/config.yml]:
```suggestion
                  -exec helm lint {} +
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -77,11 +97,91 @@ jobs:
...
+            helm plugin install https://github.com/chartmuseum/helm-push
+
+      - run:
+          name: Check versions
+          command: |
+            chart_name="$(echo "${CIRCLE_TAG}" | awk -F '/' '{print $1}')"
+            chart_version="$(echo "${CIRCLE_TAG}" | awk -F '/' '{print $2}')"
+
+            if [ ! -d "/workspace/banzai-charts/${chart_name}" ]; then
+                echo "Chart does not exists; name='${chart_name}'"
```
Comment (orymate) [inline .circleci/config.yml]:
```suggestion
                echo "Chart does not exist; name='${chart_name}'"
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -77,11 +97,91 @@ jobs:
...
+
+    steps:
+      - checkout
+
+      - run:
+          name: Setup helm repositories
+          command: |
+            helm init -c
+            helm repo remove local
+            helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
```
Comment (orymate) [inline .circleci/config.yml]: (why) do we need this?

Comment (kozmagabor) [reply .circleci/config.yml]: https://github.com/helm/charts/pull/12372/files

Comment (asdwsda) [reply .circleci/config.yml]: Required for `cadence`
```bash
$ helm dependency update cadence/
Error: no repository definition for http://storage.googleapis.com/kubernetes-charts-incubator. Please add them via 'helm repo add'
```

**Pull request merged**